packages/yourloops/pages/training/training.tsx

```diff
@@ -70,10 +70,7 @@ const style = makeStyles((theme: Theme) => {
     buttons: {
       display: 'flex',
       justifyContent: 'space-around',
-      marginTop: theme.spacing(2),
-      [theme.breakpoints.down('sm')]: {
-        justifyContent: 'space-between'
-      }
+      marginTop: theme.spacing(2)
```

Comment:
`display flex` is not usefull here, you can remove it. Also you could add a little bit more margin top